src/site/xdoc/server/config-elasticsearch.xml

```diff
@@ -116,7 +116,8 @@
       </dl>
 
       <p>
-          ElasticSearch component can be disabled. This is controlled via the <code>enabled</code> property (boolean, defaults
+          ElasticSearch component can be disabled but considered it would make search feature wont work.
```

Comment:
```suggestion
          ElasticSearch component can be disabled but consider it would make search feature to not work. In particular it will break JMAP protocol and SEARCH IMAP comment in an nondeterministic way.
```